packages/flutter/test/material/app_test.dart

```diff
@@ -143,4 +143,21 @@ void main() {
 
     expect(find.text('Home'), findsOneWidget);
   });
+
+  testWidgets('Could not implicitly use root initialRoute',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(new MaterialApp(routes: <String, WidgetBuilder>{
+      '/': (context) => new Text('route "/"')
+    }));
```

Comment:
I think you should also test that you can find the widget with text `route "/"`, otherwise this test is really only a control test that verifies nothing crashes.